### PR TITLE
LIME-989 adding contract tests for DL scenarios

### PR DESCRIPTION
**Pull request by gds-adhenderson**
### What changed

A new testing file has been added which includes scenarios and outcomes from DL CRI

### Why did it change

To test against what has been agreed, and what our VC should contain, and ensure we don't deviate from it.

### Issue tracking

- [LIME-989](https://govukverify.atlassian.net/browse/LIME-989)

### Environment variables or secrets

[LIME-989]: https://govukverify.atlassian.net/browse/LIME-989?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/pact/IssueCredentialHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+        documentCheckResultItem.setDocumentNumber("PARKE610112PBFGH");
+        documentCheckResultItem.setCheckMethod("data");
+        documentCheckResultItem.setIdentityCheckPolicy("published");
+        documentCheckResultItem.setActivityFrom("1982-05-23");
+        documentCheckResultItem.setValidityScore(0);
+        documentCheckResultItem.setIssuedBy("DVLA");
+        documentCheckResultItem.setIssueNumber("12");
+        documentCheckResultItem.setIssueDate("1982-05-23");
+        documentCheckResultItem.setContraIndicators(List.of("D02"));
+        documentCheckResultItem.setActivityHistoryScore(1);
```
Comment (gds-jbentham) [inline lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/pact/IssueCredentialHandlerTest.java:265]: Should activity history score be 1 in this scenario?

Comment (gds-adhenderson) [reply lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/pact/IssueCredentialHandlerTest.java:265]: I believe this was the score provided by Dan for this scenario, do you believe it should be something else?

Comment (gds-adhenderson) [reply lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/pact/IssueCredentialHandlerTest.java]: Discussed with Dom, and updated to make this 0

File: lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/pact/utils/PreLambdaHandler.java [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+        String[] serialize = signedJWT.serialize().split("\\.");
+
+        SignedJWT signedJWTNew =
+                new SignedJWT(
+                        jwtHeader, Base64URL.from(serialize[1]), Base64URL.from(serialize[2]));
+
+        return signedJWTNew;
+    }
+
+    private void removeNull(Map map) {
```
Comment (gds-jbentham) [inline lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/pact/utils/PreLambdaHandler.java:229]: Can this be removed and fixed the same way as fraud?

Comment (gds-adhenderson) [reply lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/pact/utils/PreLambdaHandler.java]: Updated @gds-jbentham

**Review by gds-jbentham — APPROVED** [review]

**Pull request merged**